spyder/plugins/ipythonconsole/confpage.py

```diff
@@ -113,10 +113,10 @@ def setup_page(self):
         backend_group = QGroupBox(_("Graphics backend"))
```

Comment:
@steff456, please revert the changes you made in this file because they are not necessary to solve this issue.

Besides, it would render the texts you changed to fallback to English because there would be a mismatch between the strings in our code and the translated ones.

Reply:
In fact, by changing only the other file the error persists.

When I debug the error, the texts existed but for some reason by the string formatting the error was present. So I tried a different formatting and the bug was fixed.

Reply:
This is no fix at all because (as I told you above) by changing the string (in this case, by using `format` instead of classical string replacement), you're making the current Chinese translation of that string outdated, which makes it fallback to English:

![Selección_007](https://user-images.githubusercontent.com/365293/59521321-4f837a80-8ecc-11e9-8e5f-4fba4a8071e5.png)

So, please revert these changes and fix the other errors that appear.